### DISCUSSION TITLE
Applied for student pack

### DISCUSSION THREAD

**Discussion by Muhammad-Atif-K** [AammarTufail/python_ka_chilla2023#8] [Q&A]
I have applied for a student pack. Now how to know that  my request for student pack is approved. screen shot is attached.
<img width="957" alt="Screenshot 2023-01-03 233538" src="https://user-images.githubusercontent.com/121885523/210419982-f258e0f3-2101-41cd-b1ec-966a68433c03.png">

**Comment by usamabajwa86** [comment]
you are registered

**Comment by Masoodullah909** [comment]
> I have applied for a student pack. Now how to know that my request for student pack is approved. screen shot is attached. <img alt="Screenshot 2023-01-03 233538" width="957" src="https://user-images.githubusercontent.com/121885523/210419982-f258e0f3-2101-41cd-b1ec-966a68433c03.png">

If you have applied for the GitHub Student Developer Pack, you should have received an email confirmation that your application has been received. This email should also include instructions on how to check the status of your application and what you can expect in terms of the next steps. If you have not received an email or updates on the status of your application after a week, contact GitHub for assistance through the Help Center. Your application should be processed within a few days.
